src/sentry/tasks/post_process.py

```diff
@@ -141,16 +145,11 @@ def post_process_group(
             event = Event(
                 project_id=data["project"], event_id=data["event_id"], group_id=group_id, data=data
             )
-        elif check_event_already_post_processed(event):
+        elif event and check_event_already_post_processed(event):
             if cache_key:
                 event_processing_store.delete_by_key(cache_key)
             logger.info(
-                "post_process.skipped",
-                extra={
-                    "project_id": event.project_id,
-                    "event_id": event.event_id,
-                    "reason": "duplicate",
-                },
+                "post_process.skipped", extra={"cache_key": cache_key, "reason": "missing_cache"},
```

Comment:
@markstory I realize that the `project_id` is in the cache key, but having an explicit `project_id` param in logging might help debugging.